samples/README.md

```diff
@@ -13,7 +14,81 @@ curl -O -fsSL https://raw.github.com/ppc64le-cloud/pvsadm/master/samples/image-c
 curl -O -fsSL https://raw.github.com/ppc64le-cloud/pvsadm/master/samples/image-create-and-upload-sample/config.sh
 ```
 2. Change values in the config.sh file as per description given in file
-3. Run the script 
+3. Run the script
 ```
 ./image_create_and_upload_script.sh
-```
\ No newline at end of file
+```
+
+### Script 2: Sample script for RHEL/RHCOS/CENTOS images in PowerVS service from a Centos VM.
+The convert-upload-images-powervs script helps to create RHEL/RHCOS/CENTOS images in PowerVS services. This script creates its own CentOS vm to do all the conversion tasks.
+
+#### What this script does - High Level Steps
+1. Gets CentOS OVA image from IBM FTP server or Unicamp
+2. Upload CentOS ova image COS bucket and import to PowerVS service [pvsadm tool]
+3. Spin up a CentOS instance in PowerVS using the newly uploaded CentOS image [call Terraform code]
+5. Convert given RHEL/RHCOS/CENTOS images from the spun up CentOS instance [pvsadm tool]
+6. Upload and Import the RHEL/RHCOS/CENTOS OVA images from the CentOS instance [pvsadm tool]
+
+#### Environments supported
+1. Mac
+2. Windows-10 WSL or Cygwin
```

Comment:
Don't we want to mention Linux? The script had those checks for Linux.

Reply:
yes, validated after sending the PR.. will add Linux too